bin/circleci-install-shellcheck

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
@charlierudolph: Ah, mindless copy-and-paste. Good catch!

@allewun: Yes, you can't use it in the integrated form (`/bin/bash -e`), but separately it works well, since `set -e` is just another bash command.